test/sanity/code-smell/changelog.py

```diff
@@ -18,7 +18,7 @@ def main():
         if ext not in allowed_extensions:
             print('%s:%d:%d: extension must be one of: %s' % (path, 0, 0, ', '.join(allowed_extensions)))
 
-    cmd = ['packaging/release/changelogs/changelog.py', 'lint'] + paths
+    cmd = ['antsibull-changelog', 'lint'] + paths
```

Comment:
We should invoke the module with `-m` to make sure the correct interpreter is used:

```suggestion
    cmd = [sys.executable, '-m', 'antsibull_changelog', 'lint'] + paths
```

However, it looks like that method of invocation is not currently supported by `antsibull-changelog`.

Yes, this was an existing problem with the old script invocation, but now that we're changing it we should fix it.

Reply:
Alternatively, if the module provides a public API we could just import it, which would be even better.

Reply:
Hmm... I guess that doesn't entirely fix the problem either, because `antsibull-changelog` also uses `subprocess` to invoke `ansible-doc` which may pick up the wrong one, or the wrong (or no) python.

Reply:
https://github.com/ansible-community/antsibull-changelog/pull/12